watcher/standard/src/main/java/io/fabric8/maven/watcher/standard/SpringBootWatcher.java

```diff
@@ -70,8 +80,46 @@ public void watch(List<ImageConfiguration> configs, Set<HasMetadata> resources,
 
         new PodLogService(logContext).tailAppPodsLogs(kubernetes, getContext().getNamespace(), resources, false, null, true, null, false);
 
+        String url = getServiceExposeUrl(kubernetes, resources);
+        if (url == null) {
+            url = getPortForwardUrl(resources);
```

Comment:
Not sure.. I'd be happy if the port-forward thing didn't depend on external tools, but just on the Kubernetes client, but I didn't manage to implement it for all versions of Kubernetes. I think for HTTP requests (like the ones issued by the spring-remote tool), using the Kubernetes proxy api instead of port-forward may be a better approach that doesn't rely on oc/kubectl... for a future PR..